bound/kt/pom.xml

```diff
@@ -1,152 +1,645 @@
-        <executions>
-          <execution>
-            <id>copy-dependencies</id>
-            <phase>package</phase>
-            <goals>
-              <goal>copy-dependencies</goal>
-            </goals>
-            <configuration>
-              <outputDirectory>${project.build.directory}/lib</outputDirectory>
-              <includeScope>runtime</includeScope>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-assembly-plugin</artifactId>
-        <version>3.3.0</version>
-        <configuration>
-          <descriptorRefs>
-            <descriptorRef>jar-with-dependencies</descriptorRef>
-          </descriptorRefs>
-        </configuration>
-        <executions>
-          <execution>
-            <id>make-assembly</id>
-            <phase>package</phase>
-            <goals>
-              <goal>single</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
-    </plugins>
-  </build>
+
+    <!-- Model Information -->
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- Artifact Information  -->
+    <groupId>xyz.block</groupId>
+    <artifactId>tbdex</artifactId>
+    <version>0.0.0-main-SNAPSHOT</version>
+    <name>tbDEX SDK for the JVM</name>
+    <url>https://developer.tbd.website</url>
+    <description>tbDEX SDK for the JVM</description>
+    <inceptionYear>2024</inceptionYear>
+
+    <!-- SCM -->
+    <scm>
+        <connection>scm:git:git://github.com/TBD54566975/tbdex-rs.git</connection>
+        <!-- This has to be HTTPS, not git://, for maven-release-plugin to do AUTH correctly -->
+        <developerConnection>scm:git:https://github.com/TBD54566975/tbdex-rs.git</developerConnection>
+        <url>https://github.com/TBD54566975/tbdex-rs</url>
+        <tag>HEAD</tag>
+    </scm>
+
+    <!-- Developers -->
+    <developers>
+        <developer>
+            <id>TBD54566975</id>
+            <name>Block, Inc.</name>
+            <email>releases@tbd.email</email>
+        </developer>
+    </developers>
+
+    <!-- Issues -->
+    <issueManagement>
+        <system>github</system>
+        <url>https://github.com/TBD54566975/tbdex-rs/issues</url>
+    </issueManagement>
+
+    <!-- Licenses -->
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <!-- Properties -->
+    <properties>
+        <!-- Project properties -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <!--
+
+        //TODO Review with Kendall and Neal if we can move this to Java17.
+
+        If so we need to update the workflows in the Kotlin CI to use that version too.
+
+        -->
+        <kotlin.jvm.target>11</kotlin.jvm.target>
```

Comment:
I don't see why not to move it to 17, but I do have a faint memory with web5-kt we had to keep it 11...

Reply:
Yup, we're prioritizing getting to the bottom of this, but doesn't need to block things here

Reply:
Yes, by release I'd like us to move to `17` but this keeps the train running for now.